src/quo/previews/reacts.cljs

```diff
@@ -0,0 +1,71 @@
+(ns quo.previews.reacts
+  (:require [reagent.core :as reagent]
+            [quo.core :as quo]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo.react-native :as rn]
+            [quo.design-system.colors :as colors]))
+
+(def reaction-styling
+  {:display "flex"
+   :flex-direction "row"
+   :padding-vertical 8
+   :padding-horizontal 8
+   :margin-top 25
+   :border-radius 10
+
+   :border-width 1})
+
+(defn render-react
+  "Add your emoji as a param here"
+  [{:keys [emoji clicks dark? neutral?]}]
+  (let [text-color (if dark? "white" "black")
+        clicks-positive? (pos-int? @clicks)]
+    [rn/touchable-opacity {:on-press #(swap! clicks inc)
+                           :style (merge reaction-styling
+                                         {:border-color (if dark? "white" "black")
+                                          :background-color   (if dark?
+                                                                (if neutral? "#192438" "black")
```

Comment:
You can use the colors from `src/quo2/foundations/colors.cljs`, if you inspect in figma, you should see the name of the color and there should be a matching one in that file.
Also just by looking at the designs and the screenshots, seems like it's a bit off (for example there are no black borders on the designs, text color in dark mode is different), probably worth going over them again to check that they are matching, if you want we can go through them together and also @pedro-et can help out with how we use figma and how to find your way around components etc

Reply:
Let's have a chat on it then today